packages/flutter/lib/src/rendering/view.dart

```diff
@@ -7,6 +7,7 @@ import 'dart:ui' as ui show FlutterView, Scene, SceneBuilder, SemanticsUpdate;
 
 import 'package:flutter/foundation.dart';
 import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
```

Comment:
This is a layer violation: The rendering layer cannot import the widget layer.

Reply:
I changed to flutter/scheduler.dart